contrib/Detached.Mappers.EntityFramework.Contrib.SysTec/ComplexDbContext.cs

```diff
@@ -51,6 +52,13 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
                     //    .Value(nameof(Customer), typeof(Customer))
                     //    .Value(nameof(Government), typeof(Government))
                     //    .Value(nameof(SubGovernment), typeof(GovernmentLeader));
+
+                    options.Type<OrganizationBaseDTO>().Abstract();
+
+                    options.Type<OrganizationBaseDTO>()
```

Comment:
sorry, didn't like InheritanceDiscriminator solution very much,
added a regular discriminator to the DTO, of course, values should match the ones in the entity and the discriminator property must exist in DTO, which it does in your example (even when this should't be a must for DTOs) 
notice that if discriminator is present, the projection expression is not evaluated at DB side, and the entire entity is materialzed, there is no way for the system to select specific fields in a row basis

Reply:
needed for 13 to run

Reply:
The solution you provided is my favourite, too. It is better than the attribute.

If you have different DTOs to Entity Mapping, than you have to use profiles, right?